proposals/0005-objective-c-name-translation.md

```diff
@@ -569,8 +569,8 @@ func <b>with</b>Matrix(_: CGAffineTransform) -> UIFontDescriptor
 </pre>
 
 If we instead began by pruning `SymbolicTraits` from the tail of
-the first method name, the prohibition against creating `absurdly
-vacuous names`_ would prevent us from pruning "`fontDescriptorWith`"
+the first method name, the prohibition against creating "absurdly
+vacuous names" would prevent us from pruning "`fontDescriptorWith`"
```

Comment:
I don't think we even need double quotes here.  It is just a link to a section that is not included in the document.